dist/marionette.toolkit.esm.js

```diff
@@ -278,6 +278,10 @@ var ChildAppsMixin = {
    * @method _startChildApp
    */
   _startChildApp: function _startChildApp(childApp, options) {
+    if (!childApp) {
```

Comment:
What about if we threw the error in `startChildApp` so we could throw it with `appName`?  I _think_ that'd work without issues elsewhere and it'd provide a more useful error.